Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -187,4 +265,15 @@ public void SetResetParameters()
     {
         SetTorsoMass();
     }
+
+    private void OnDrawGizmosSelected()
+    {
+        if (Application.isPlaying)
```

Comment:
I __think__ this can only be used in the Editor.

Reply:
I think i may move this logic to a separate script in a future PR

Reply:
Does in run in a built executable ?

Reply:
The OnDrawGizmo callbacks are not executed in builds, only the editor. The builds have worked without any errors.